truss/templates/control/control/server.py

```diff
@@ -37,21 +35,29 @@ def _identify_python_executable_path() -> str:
                 python_executable_path,
                 f"{inf_serv_home}/inference_server.py",
             ],
-            "control_server_host": "*",
+            "control_server_host": "0.0.0.0",
             "control_server_port": CONTROL_SERVER_PORT,
             "inference_server_port": INFERENCE_SERVER_PORT,
         }
     )
 
-    # Perform inference server startup flow in background
-    Thread(target=inference_server_startup_flow, args=(application,)).start()
-
-    application.logger.info(
+    application.state.logger.info(
         f"Starting live reload server on port {CONTROL_SERVER_PORT}"
     )
-    server: Union[BaseWSGIServer, MultiSocketServer] = create_server(
+
+    cfg = uvicorn.Config(
         application,
-        host=application.config["control_server_host"],
-        port=application.config["control_server_port"],
+        host=application.state.control_server_host,
+        port=application.state.control_server_port,
+        workers=1,
     )
-    server.run()
+    cfg.setup_event_loop()
+
+    max_asyncio_workers = min(32, utils.cpu_count() + 4)
```

Comment:
@pankajroark is it obvious how to change this worker scheme that you were talking about this morning? Should we just set this to 1?

Reply:
yeah, we can set this to 1